sui/amount-utils.js

```diff
@@ -0,0 +1,16 @@
+const { ethers } = require('ethers');
+
+// Convert formatted amount to atomic units (e.g. 1000000000). Default decimals is 9 for SUI
+function getAtomicAmount(amount, decimals = 9) {
```

Comment:
```suggestion
function getUnitAmount(amount, decimals = 9) {
```
we usually refer to it as unit